src/api/production.yml

```diff
@@ -42,6 +48,7 @@ services:
     labels:
       # Add gzip compression too
       - 'traefik.http.routers.image.middlewares=gzip_compress,strip_api_prefix'
+      - 'traefik.http.routers.image.tls=true'
```

Comment:
So we have to explicitly enable tls on all these services?

Reply:
Yes, I thought it'd be a good idea to keep it flexible and only enabling it for specific services.
If we know that we want all the services to use tls, maybe I should enable it by default for all services?

Reply:
I think that in the context of this file (btw, I'm refactoring it a bit in https://github.com/Seneca-CDOT/telescope/pull/1840 to separate our traefik+microservices from dev deps), anything we enable with traefik should be tls, no?  Especially if we disable :80, there's no point doing traefik for a container if we don't expose it via tls.